packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/virtualmachine/dedicatedCloud-datacenter-virtualmachine.html

```diff
@@ -0,0 +1,72 @@
+<div class="dedicatedCloud-datacenter-virtualmachines">
+    <h2 data-translate="virtualmachine_title"></h2>
+    <div class="text-center" data-ng-if="$ctrl.loading">
+        <oui-spinner></oui-spinner>
+    </div>
+    <div data-ng-if="!$ctrl.loading">
+        <oui-datagrid data-rows-loader="$ctrl.loadVirtualMachines($config)">
+            <oui-datagrid-column
+                data-title=":: 'virtualmachine_grid_name' | translate"
+            >
+                <span data-ng-bind="$row.name"></span>
+            </oui-datagrid-column>
+
+            <oui-datagrid-column
+                data-title=":: 'virtualmachine_grid_guest_os' | translate"
+            >
+                <span data-ng-bind="$row.guestOsFamily"></span>
```

Comment:
You can use `data-property` : https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-datagrid-webcomponent--remote-data